modules/mapper-extras/src/yamlRestTest/resources/rest-api-spec/test/rank_feature/10_basic.yml

```diff
@@ -198,3 +255,108 @@ setup:
 
   - match:
       hits.hits.1._id: "1"
+
+---
+"Non positive null_vallue":
+
+  - do:
+      catch: bad_request
+      indices.create:
+        index: test1
```

Comment:
You should create some other index different from from index created in `setup`, or you can avoid `setup` section at all in the new file.